pkg/networkmanager/wireguard.jsx

```diff
@@ -73,7 +73,12 @@ export function WireGuardDialog({ settings, connection, dev }) {
     const [listenPort, setListenPort] = useState(settings.wireguard.listen_port);
     const [addresses, setAddresses] = useState(addressesToString(settings.ipv4.addresses));
     const [dialogError, setDialogError] = useState("");
-    const [peers, setPeers] = useState(settings.wireguard.peers.map(peer => ({ ...peer, allowedIps: peer.allowedIps?.join(",") ?? '' })));
+    const [peers, setPeers] = useState(settings.wireguard.peers.map(peer => ({
+        ...peer,
+        allowedIps: peer.allowedIps?.join(",") ?? '',
+        persistentKeepalive: peer.persistentKeepalive?.toString() ?? '',
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19521-20231026-124818-30ae48de-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)